packages/server/src/cli.ts

```diff
@@ -236,12 +236,10 @@ const bold = (text: string | number): string | number => {
 	});
 
 	if (options.noAuth || options.allowHttp) {
-		logger.info("Starting webserver...", field("host", "127.0.0.1"), field("port", options.port));
-		app.server.listen(options.port, "127.0.0.1");
-	} else {
-		logger.info("Starting webserver...", field("host", options.host), field("port", options.port));
-		app.server.listen(options.port, options.host);
+		logger.warn("0.0.0.0 is accessible to any device on your network");
```

Comment:
I think we should not generate this message when the string `host` is not `0.0.0.0`.

Reply:
updated